### PR TITLE
docs/sapphire: Add precompile descriptions

### PR DESCRIPTION
**Pull request by jberci**

### CODE REVIEW

**Review by nhynes — APPROVED** [review]
Thanks, this is great. Let's just make sure the gas costs match the time costs. Also, it'll be really great to have this EVM assembly bundled into a `Sapphire.sol` library for people to use.

File: docs/dapp/sapphire/precompiles.md
Comment (matevz) [inline docs/dapp/sapphire/precompiles.md]: This is rendered in the doc cards so it needs to be short and concise, like:

```suggestion
description: Additional Sapphire precompiles for encryption and confidentiality
```

File: docs/dapp/sapphire/precompiles.md
Comment (matevz) [inline docs/dapp/sapphire/precompiles.md]:
```suggestion
These can be called in the same way as other precompiles by
```

File: docs/dapp/sapphire/precompiles.md
Comment (matevz) [inline docs/dapp/sapphire/precompiles.md]:
```suggestion
memory region, with each chunk of data padded to 32 bytes as usual.
```

File: docs/dapp/sapphire/precompiles.md
Comment (matevz) [inline docs/dapp/sapphire/precompiles.md]: Where is this Sapphire repo? I cannot seem to find any solidity code in https://github.com/oasisprotocol/sapphire-paratime/. Please add a link to `library/Sapphire.sol` file.

**Review by matevz — APPROVED** [review]
Some minor details. However, this PR needs to wait until x25519 and deoxys actually land on Testnet right?

File: docs/dapp/sapphire/precompiles.md
Comment (kostko) [reply docs/dapp/sapphire/precompiles.md]: https://github.com/oasisprotocol/sapphire-paratime/pull/44

Comment (jberci) [reply docs/dapp/sapphire/precompiles.md]: I'll edit the text here once that PR is merged and the path is final.

**Comment by kostko** [comment]
We should probably merge now that the Testnet has been updated to include these.

**Pull request merged**
